ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -70,19 +70,33 @@ public static void setUp() {
         createTestData();
     }
 
-    private static void CreateTableAndMapping() {
+    @AfterAll
+    public static void tearDown() {
         try {
             queryClient.execute(databaseName, String.format(".drop table %s ifexists", tableName));
-            Thread.sleep(1000);
-            queryClient.execute(databaseName, String.format(".create table %s %s", tableName, tableColumns));
+        } catch (Exception ex) {
+            Assertions.fail("Failed to drop table", ex);
+        }
+    }
+
+    private static void CreateTableAndMapping() {
+        KustoOperationResult result;
+        try {
+            result = queryClient.execute(databaseName, String.format(".drop table %s ifexists", tableName));
+        } catch (Exception ex) {
+        }
+        try {
+            Thread.sleep(2000);
+            result = queryClient.execute(databaseName, String.format(".create table %s %s", tableName, tableColumns));
         } catch (Exception ex) {
             Assertions.fail("Failed to drop and create new table", ex);
         }
 
         resourcesPath = Paths.get(System.getProperty("user.dir"), "src", "test", "resources").toString();
         try {
+            Thread.sleep(2000);
             String mappingAsString = new String(Files.readAllBytes(Paths.get(resourcesPath, "dataset_mapping.json")));
-            queryClient.execute(databaseName, String.format(".create table %s ingestion json mapping '%s' '%s'",
+            result = queryClient.execute(databaseName, String.format(".create table %s ingestion json mapping '%s' '%s'",
```

Comment:
I added a few of these in some system tests. They're useful for debugging, and it's NBD to leave them in a system test class instead of having to constantly remove and re-add.